tests/pure_directaccess/pure_test.go

```diff
@@ -62,6 +63,10 @@ func createCloudsnapCredential() {
 		TimeBeforeRetry: k8s.DefaultRetryInterval,
 		Sudo:            true,
 	})
+	// if the cloudsnap credentials already exist, just leave them there
+	if err != nil && !strings.Contains(err.Error(), "already exist") {
```

Comment:
```suggestion
	if err != nil && !strings.Contains(err.Error(), "cloudsnap credentials already exist") {
```

Reply:
I wanted to leave the error message broad in case that message happened to change. I figured "already exist" would be a good catch-all